src/VisualStudio/Core/Def/Implementation/ChangeSignature/ChangeSignatureDialogViewModel.cs

```diff
@@ -488,8 +687,8 @@ string ModifierText(string @out = default, string @ref = default, string @in = d
                             return @params ?? string.Empty;
                         }
 
-                        if (_changeSignatureDialogViewModel._thisParameter != null &&
-                            Equals(ParameterSymbol, _changeSignatureDialogViewModel._thisParameter.ParameterSymbol))
+                        if (changeSignatureDialogViewModel._thisParameter != null &&
```

Comment:
We can't just write changeSignatureDialogViewModel._thisParameter == this?

Reply:
Assumed removing and re-creating this parameter in the future, the current comparison looks safer.

---
In reply to: [367198868](https://github.com/dotnet/roslyn/pull/38748#discussion_r367198868) [](ancestors = 367198868)